.circleci/config.yml

```diff
@@ -287,14 +300,21 @@ commands:
             - go-cache-macos-v2-{{ checksum "go.mod" }}
       - restore_cache:
           keys:
-            - llvm-source-11-macos-v1
+            - llvm-source-11-macos-v2
       - run:
           name: "Fetch LLVM source"
-          command: make llvm-source
+          command: |
+            if [ ! -e llvm-project ]
+            then
+              make llvm-source
+            fi
```

Comment:
This change is unnecessary: `make llvm-source` will exit immediately if the llvm-project directory exists.

Reply:
You are correct.
I fixed it and force-pushed it.